week5/5.3/whisper.py

```diff
@@ -0,0 +1,35 @@
+"""
+whisper.
+By: Eliad Karni.
+The feature decode messages which contained in the file "logo.jpg".
+Used the help of the websites:
+Binary read -> https://pythonguides.com/python-read-a-binary-file/
+Ignore decode errors -> https://docs.python.org/3/library/codecs.html
+"""
+
+
+def get_a_whisper() -> str:
+    """
+    The function is a iterate generator which decode a whisper from the file "logo.jpg".
```

Comment:
Nice doc, but you are missing some things  - read this and fix the docs - 
https://stackoverflow.com/questions/2776829/difference-between-pythons-generators-and-iterators

Reply:
I think I understood what you meant. and fixed it.